staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -141,16 +183,21 @@ func sortByPriority(specs []openAPISpecInfo) {
 
 // buildOpenAPISpec aggregates all OpenAPI specs.  It is not thread-safe.
 func (s *openAPIAggregator) buildOpenAPISpec() (specToReturn *spec.Swagger, err error) {
-	specToReturn, err = aggregator.CloneSpec(s.inProcessDelegatesOpenAPISpec)
+	firstDelegateName := fmt.Sprintf(localDelegateChainNamePattern, 0)
+	localDelegateSpec, exists := s.openAPISpecs[firstDelegateName]
+	if !exists {
+		return nil, fmt.Errorf("localDelegate spec is missing")
+	}
+	specToReturn, err = aggregator.CloneSpec(localDelegateSpec.spec)
```

Comment:
see my other comment about using the delegation position as priority for the local services. Then we can get rid of the special handling here.